code/modules/admin/verbs/mapping.dm

```diff
@@ -56,12 +47,16 @@ GLOBAL_VAR_INIT(intercom_range_display_status, 0)
 	else
 		GLOB.camera_range_display_status = 1
 
-	for(var/obj/effect/debugging/camera_range/C in world)
-		qdel(C)
+	for(var/obj/effect/debugging/marker/M in world)
+		qdel(M)
 
 	if(GLOB.camera_range_display_status)
 		for(var/obj/machinery/camera/C in GLOB.cameranet.cameras)
-			new/obj/effect/debugging/camera_range(C.loc)
+			for(var/turf/T in orange(7, C))
+				var/obj/effect/debugging/marker/F = new/obj/effect/debugging/marker(T)
+				if(!(F in view(7, C.loc)))
```

Comment:
Cache the `view(7, C.loc)` out of the `orange(7, C)` loop to avoid redundancy.

Reply:
This is a verb used by people on local test servers when mapping and only invokes once (its not a constant ticking loop)